File: iCyption2022/CftTest/Balap/solution.py
finish, start_a, start_b, speed_a, speed_b = map(int, input().split())

step_a = ((finish - start_a) // speed_a) + int((finish - start_a) % speed_a != 0)
step_b = ((finish - start_b) // speed_b) + int((finish - start_b) % speed_b != 0)

if (step_a < step_b) :
    print("okawai koto")
elif (step_a > step_b) :
    print("like a man")
else :
    print("dokoisho")

File: iCyption2022/CftTest/Balap/solution.cpp
#include <bits/stdc++.h>

using namespace std;

#define ll long long
#define ld long double

int main() {
    ll finish, start_a, start_b, speed_a, speed_b;
    cin >> finish >> start_a >> start_b >> speed_a >> speed_b;

    ll step_a = ((finish - start_a) / speed_a) + ((finish - start_a) % speed_a != 0);
    ll step_b = ((finish - start_b) / speed_b) + ((finish - start_b) % speed_b != 0);

    if (step_a < step_b) {
        cout << "okawai koto" << endl;
    } else if (step_a > step_b) {
        cout << "like a man" << endl;
    } else {
        cout << "dokoisho" << endl;
    }
}